cli/src/lib.rs

```diff
@@ -190,6 +204,13 @@ pub enum Command {
     Publish {
         /// The name of the program to publish.
         program: String,
+        /// Docker image to use.
```

Comment:
These fields aren't plumbed through to the build server so this won't work as intended. Probably best to remove from the `publish` command for now to minimize confusion.

Reply:
Ah good catch, I misunderstood this part.  I changed this to use the defaults